src/app/components/media/Similarity/MediaSuggestionsComponent.js

```diff
@@ -415,49 +669,284 @@ const MediaSuggestionsComponent = ({
+                        openBulkRejectDialog();
+                      }}
+                      disabled={disableAcceptRejectButtons}
+                      className={`${disableAcceptRejectButtons ? classes.disabled : ''} ${classes.reject} similarity-media-item__reject-relationship`}
+                    >
+                      <RejectIcon fontSize="large" />
+                    </IconButton>
+                  </Tooltip>
+                </Grid>
+                <Grid item>
+                  <Tooltip title={<FormattedMessage id="mediaSuggestionsComponent.bulkSpam" defaultMessage="Mark all medias on this page as spam" description="Tooltip for a button that is an octagon with an exclamation mark. Pressing this button causes all visible media items on the page to be marked as 'spam' and removed from the suggestions list." />}>
+                    <IconButton
+                      onClick={() => {
+                        openBulkSpamDialog();
+                      }}
+                      disabled={disableAcceptRejectButtons}
+                      className={`${disableAcceptRejectButtons ? classes.disabled : ''} ${classes.spamTrash} similarity-media-item__reject-relationship`}
+                    >
+                      <SpamIcon fontSize="large" />
+                    </IconButton>
+                  </Tooltip>
+                </Grid>
+                <Grid item>
+                  <Tooltip title={<FormattedMessage id="mediaSuggestionsComponent.bulkTrash" defaultMessage="Send all medias on this page to trash" description="Tooltip for a button that is a trash bin. Pressing this button causes all visible media items on the page to be sent to the 'trash' folder and removed from the suggestions list." />}>
+                    <IconButton
+                      onClick={() => {
+                        openBulkTrashDialog();
+                      }}
+                      disabled={disableAcceptRejectButtons}
+                      className={`${disableAcceptRejectButtons ? classes.disabled : ''} ${classes.spamTrash} similarity-media-item__reject-relationship`}
+                    >
+                      <TrashIcon fontSize="large" />
+                    </IconButton>
+                  </Tooltip>
+                </Grid>
+              </Grid>
+              <IconButton className={classes.helpIconContainer} onClick={handleHelp}>
                 <HelpIcon className={classes.helpIcon} />
               </IconButton>
             </Box>
-            <Typography variant="body2">
-              <FormattedMessage
-                id="mediaSuggestionsComponent.question"
-                defaultMessage="Are the medias below a good match for the claim and medias on the left?"
-                description="Subtitle for similarity matching widget"
-              />
-            </Typography>
             <Box display="flex" alignItems="center">
               <>
+                <SuggestedMediaDialogComponent
```

Comment:
This div contains six different dialog box components. Ultimately this needs to be turned into a CDS dialog box component and it should be six lines of code here like

```
<ConfirmationDialog title={foo} confirmButtonText={bar} />
<ConfirmationDialog title={baz} confirmButtonText={biz} />
```

Instead of componentizing this early, I figured I would copy/paste code here and make a bookmark to come back when it's time to work on developing the confirmation dialog component.